.ci/Jenkinsfile

```diff
@@ -33,6 +33,7 @@ pipeline {
      */
     stage('Checkout') {
       steps {
+        pipelineManager([ cancelPreviousRunningBuilds: [ when: 'PR' ] ])
```

Comment:
This line will kill previous executions for a same PR, avoid consuming CI resources unnecessarily

Reply:
@mtojek because we added this behaviour in a separate commit, we can rever it if we don't like it